resources/views/users/edit.blade.php

```diff
@@ -0,0 +1,122 @@
+@extends('layouts.master')
+@section('main-content')
+    @if(!isset($user))
+        <h1>User not found</h1>
+    @else
+        <div class="row">
+
+            @include('flash::message')
+
+            <form class="form-horizontal" enctype="multipart/form-data" action="{{ route('users.update', $user->id) }}" method="POST">
+                <input type="hidden" name="_method" value="PUT">
```

Comment:
You can use `method_field()` to add input hidden `_method`
https://laravel.com/docs/5.4/helpers#method-method-field